lib/mix/lib/mix/tasks/test.ex

```diff
@@ -434,7 +434,7 @@ defmodule Mix.Tasks.Test do
         files
         |> Enum.map(&Path.expand/1)
         |> Enum.map(&Path.relative_to_cwd/1)
-        |> Enum.split_with(&String.starts_with?(&1, "apps/"))
+        |> Enum.split_with(&String.contains?(&1, "apps/"))
```

Comment:
Unfortunately this check can have also positives, for example, if you have a folder called "company-apps". We will need to take a look into other ways to address this.